tests/conftest.py

```diff
@@ -1774,9 +1774,21 @@ def label_registry(hass: HomeAssistant) -> lr.LabelRegistry:
 
 
 @pytest.fixture
-def service_calls(hass: HomeAssistant) -> list[ServiceCall]:
-    """Track calls to a mock service."""
-    return async_mock_service(hass, "test", "automation")
+def mocked_service() -> tuple[str, str]:
+    """Fixture to allow overriding mocked service."""
+    return ("test", "automation")
```

Comment:
Not sure why this is added?

Let's just override all service calls for this mock instead.